pkg/extensions/search/resolver.go

```diff
@@ -27,6 +27,11 @@ import (
 	"zotregistry.io/zot/pkg/storage"
 ) // THIS CODE IS A STARTING POINT ONLY. IT WILL NOT BE UPDATED WITH SCHEMA CHANGES.
 
+var (
```

Comment:
Do not redefine errors from `errors/errors.go`. Import from there if you really need them.

Reply:
I do not see how these would seem fitting!?
```golang
ErrUnauthorizedAccess      = errors.New("cli: unauthorized access. check credentials")
ErrBadUser                 = errors.New("ldap: non-existent user")
```